llarp/handlers/tun.cpp

```diff
@@ -199,42 +163,27 @@ namespace llarp
           return false;
       }
 
-      std::string ifname = conf.m_ifname;
-      if (ifname.empty())
+      m_IfName = conf.m_ifname;
+      if (m_IfName.empty())
```

Comment:
this same `if` block is in handlers/exit.cpp as well, perhaps could be more DRY.

Reply:
there is no common base class they share that makes that possible

Reply:
aight.  perhaps that's something to consider, idk.

Reply:
or rather, maybe some helper class or method that does this bit.  at any rate, not a big deal for now.